### DISCUSSION TITLE
Is it safe to delete the default Samsung "My Files" App and install this?

### DISCUSSION THREAD

**Discussion by Letseatsomepizza** [TeamAmaze/AmazeFileManager#4070] [Q&A]
Going to delete the original unremovable app **AFTER** I install Amaze of course, but should I? Is it safe for my phone? Does Amaze depend on the default Samsung app?

**Comment by VishnuSanal** [comment]
> Going to delete the original unremovable app **AFTER** I install Amaze of course, but should I? Is it safe for my phone? Does Amaze depend on the default Samsung app?

No, AFAIK. @TranceLove please verify. :)
